code/edge/build.gradle

```diff
@@ -74,6 +81,11 @@ spotless {
         endWithNewline()
         licenseHeaderFile "../../config/formatter/adobe.header.txt"
     }
+    kotlin {
+        target "src/*/java/**/*.kt"
+        ktlint('0.41.0')
```

Comment:
- core and other extensions use 0.47.0 for ktlint
- also don't forget to include the kotlin classes to the coverage report for both unit and functional tests